docs/developing-locust.rst

```diff
@@ -11,19 +11,22 @@ Install Locust for development
 
 Fork Locust on `GitHub <https://github.com/locustio/locust/>`_ and then run
 
+.. note::
+    To build the Locust web UI, you must have `node` and `yarn` installed - see the *Making changes to Locust's Web UI* section below
+
 .. code-block:: sh
 
     # clone the repo
     $ git clone git://github.com/<YourName>/locust.git
 
     # install the poetry build system
-    $ pip3 install poetry 
+    $ python -m pip install poetry 
 
     # install the dynamic versioning plugin for poetry
-    $ pip3 -m poetry self add "poetry-dynamic-versioning[plugin]"
+    $ python -m poetry self add "poetry-dynamic-versioning[plugin]"
```

Comment:
What will happen if this is not present? Can we make it optional, just to make the install easier for people? (maybe not now)

Reply:
In terms of the plugin?

The default versioning will be used, which is not that big a deal really, we could omit this from this section of the docs if it's easier 👍 

You'll just end up with something like `0.0.1dev1` or similar as the package version, not a deal breaker if you're just wanting to build a dev version of the project

What do you think?